MathlibTest/Util/Qq.lean

```diff
@@ -0,0 +1,49 @@
+import Mathlib.Util.Qq
+import Mathlib.Data.Finset.Basic
+
+
+open Qq Lean Elab Meta
+
+section mkSetLiteralQ
+
+/--
+info: {1, 2, 3}
+-/
+#guard_msgs in
+run_cmd do
+  let β := q(Finset ℕ)
+  let elems :=  [q(1), q(2), q(3)]
+  Lean.logInfo <| ← Command.liftTermElabM (ppExpr (mkSetLiteralQ β elems))
```

Comment:
```suggestion
  Lean.logInfo m!"{mkSetLiteralQ β elems}"
```
etc. I think you could just inline `β` at this point.

Reply:
Or for an even short test:
```lean
#check by_elab return mkSetLiteralQ q(Finset ℕ) [q(1), q(2), q(3)]
```

Reply:
Nice! That is indeed a lot cleaner.